setup.py

```diff
@@ -87,7 +84,11 @@ def package_files(directory):
 setup(
     name='cocotb',
     cmdclass={'build_ext': build_ext},
-    version=__version__,  # noqa: F821
+    use_scm_version={
+        'write_to': 'cocotb/_version.py',
```

Comment:
Stylistically, I sometimes like using
```suggestion
    use_scm_version=dict(
        write_to='cocotb/_version.py',
```
just to make the nested arguments resemble the non-nested ones.